### ISSUE TITLE
Please credit the repo that you’ve (possibly) referred to

### ISSUE THREAD

**Issue by seungwonpark** [JiaminRen/RandWireNN#2]
Hi, I’m author of https://github.com/seungwonpark/RandWireNN.

[One SO question](https://stackoverflow.com/questions/55566291/implementation-of-exploring-randomly-wired-neural-networks-for-image-recognition) brought me here. At first, I was excited to see another implementation of RandWireNN. However, I’ve observed that much of your code resembles my implementation(plus, you even emailed me last Saturday), and was disappointed to see that there are no statement referencing my repo.

I’m not saying that this repo is just copy-and-paste of mine. There are several useful features that are not included in mine: multi-GPU support and tool for ImageNet data loading, and so on.

I would like to kindly ask you to add reference to my repository, if you have referred it.

**Comment by JiaminRen** [comment]
Are you kidding me? I finished the code as early as April 5. And most code comes from my previous work, SN(https://github.com/switchablenorms/Switchable-Normalization).Besides, I have been running the experiment for more than three days. Now it has run 52 epochs, and the top-1 accuracy has reached 70%. I also saw your code implementation. It's totally different from the generation of graphs to the building of models.I emailed you because I saw you are also working on this paper, so I want to talk more about it.
